### PR TITLE
CHANGELOG for 1.7.0

### PR DESCRIPTION
**Pull request by angelozerr**
CHANGELOG for 1.7.0

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
 # Quarkus Tools for Visual Studio Code Changelog
 
+## 1.7.0 (September 21, 2020)
+
+Since `1.7.0`, vscode-quarkus becomes an extension of [vscode-microprofile](https://github.com/redhat-developer/vscode-microprofile), see [CHANGELOG of vscode-microprofile](https://github.com/redhat-developer/vscode-microprofile/CHANGELOG.md#010-september-14-2020)
```
Comment (fbricon) [inline CHANGELOG.md]: vscode-microprofile will be automatically installed alongside vscode-quarkus.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+
+### Enhancements
+
+ * Notification when propertiesLanguageMismatch is forceQuarkus and user tries to set languageId to something else. See [#285](https://github.com/redhat-developer/vscode-quarkus/issues/285).
```
Comment (fbricon) [inline CHANGELOG.md]: when the `quarkus.tools.propertiesLanguageMismatch` setting is set to `forceQuarkus`,

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,33 @@
...
+
+### Build
+
+ * Contribute lsp4mp.ls jars. See [#290](https://github.com/redhat-developer/vscode-quarkus/issues/290).
+
+### Other
+
+ * Define Quarkus document selector and TextMate grammar. See [#294](https://github.com/redhat-developer/vscode-quarkus/pull/294).
+ * Fix links in README. See [#287](https://github.com/redhat-developer/vscode-quarkus/pull/287).
+ * Build quarkus jdt extension. See [#276](https://github.com/redhat-developer/vscode-quarkus/pull/276).
```
Comment (fbricon) [inline CHANGELOG.md]: that's a `Build` item

**Review by fbricon — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (angelozerr) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (angelozerr) [reply CHANGELOG.md]: updated

File: CHANGELOG.md (hunk 3 above)
Comment (angelozerr) [reply CHANGELOG.md]: moved

**Review by fbricon — APPROVED** [review]

**Pull request merged**